pkg/rabbit/service.go

```diff
@@ -143,6 +143,32 @@ func (r *Rabbit) ReconcileBinding(ctx context.Context, args *BindingArgs) (Resul
 	}, nil
 }
 
+func (r *Rabbit) ReconcileBrokerDLXPolicy(ctx context.Context, args *QueueArgs) (Result, error) {
+	logging.FromContext(ctx).Infow("Reconciling broker dead letter policy", zap.String("name", args.Name))
```

Comment:
`ReconcileQueue` also reconciles a policy if the DLXName is defined. Does it make sense to make this a generic `ReconcilePolicy` and have the trigger controller use that as well?

Reply:
yep sure thing